File: Beecrowd/Java/1116 - Dividing X by Y.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int X,Y,n,i;
        n= sc.nextInt();
    for(i=0;i<n;i++){
        X= sc.nextInt();
        Y= sc.nextInt();
        if(Y==0) System.out.printf("divisao impossivel\n");
        else System.out.printf("%.1f\n",(float)X/(float)Y);
    }}
    }

File: Beecrowd/C/1116 - Dividing X by Y.c
#include<stdio.h>
int main(){
    int X,Y,n,i;
    scanf("%d",&n);
    for(i=0;i<n;i++){
        scanf("%d%d",&X,&Y);
        if(Y==0) printf("divisao impossivel\n");
        else printf("%.1f\n",(float)X/(float)Y);
    }
}
